lighthouse-core/audits/byte-efficiency/uses-webp-images.js

```diff
@@ -21,10 +21,10 @@ class UsesWebPImages extends ByteEfficiencyAudit {
   static get meta() {
     return {
       name: 'uses-webp-images',
-      description: 'Serve images as WebP',
+      description: 'Serve images in superior formats',
       informative: true,
-      helpText: 'WebP provides better lossy and lossless compression than PNG or JPEG, ' +
-        'which means faster downloads and less data consumption. ' +
+      helpText: 'Image formats like JPEG 2000, JPEG XR, and WebP often provide better ' +
+        'compression than PNG or JPEG, which means faster downloads and less data consumption. ' +
         '[Learn more](https://developers.google.com/web/tools/lighthouse/audits/webp).',
```

Comment:
Shouldn't we update this documentation to explain the different formats?

Reply:
Yes, indeed we should

Reply:
Filed a bug against Kayce to update the docs for next sprint :)

Reply:
Cool cool, I also just got a brain dump from him on how I can update stuff and seems straightforward when we have changes to docs like this that aren't new material :)